tests/py/dynamo/models/test_models.py

```diff
@@ -84,6 +84,41 @@ def test_resnet18_cpu_offload(ir):
     torch._dynamo.reset()
 
 
+@pytest.mark.unit
+def test_resnet18_torch_exec_ops(ir):
```

Comment:
@unittest.skipIf(
        not importlib.util.find_spec("torchvision"), "torchvision not installed"
    )
Same here